src/main/scala/io/github/davidgregory084/ScalacOption.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 David Gregory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.davidgregory084
+
+case class ScalacOption(
```

Comment:
Maybe it would be better to just use a normal class?

Reply:
Yep you're right, we're not getting much out of `case class` here - fixed in 51a2503